Userland/Utilities/headless-browser.cpp

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright (c) 2022, Dex♪ <dexes.ttp@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Assertions.h>
+#include <AK/ByteBuffer.h>
+#include <AK/Format.h>
+#include <AK/HashTable.h>
+#include <AK/NonnullOwnPtr.h>
+#include <AK/StringBuilder.h>
+#include <AK/Types.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/EventLoop.h>
+#include <LibCore/File.h>
+#include <LibCore/Timer.h>
+#include <LibGfx/Bitmap.h>
+#include <LibGfx/Font/FontDatabase.h>
+#include <LibGfx/ImageDecoder.h>
+#include <LibGfx/PNGWriter.h>
+#include <LibGfx/Rect.h>
+#include <LibMain/Main.h>
+#include <LibWeb/Cookie/ParsedCookie.h>
+#include <LibWeb/DOM/Document.h>
+#include <LibWeb/HTML/BrowsingContext.h>
+#include <LibWeb/ImageDecoding.h>
+#include <LibWeb/Layout/InitialContainingBlock.h>
+#include <LibWeb/Loader/ResourceLoader.h>
+#include <LibWeb/Page/Page.h>
+#include <LibWeb/Painting/PaintableBox.h>
+#include <LibWeb/WebSockets/WebSocket.h>
+
+class HeadlessBrowserPageClient final
+    : public Web::PageClient {
```

Comment:
no line break here, only for multiple inheritance

Reply:
It was due to the old behavior which used multiple inheritence, but that's not the case so, fixed!